core/src/impl/Kokkos_GraphImpl.hpp

```diff
@@ -37,6 +37,12 @@ struct is_graph_capture<
            Kokkos::Impl::is_specialization_of_v<T, GraphNodeCaptureImpl>>>
     : public std::true_type {};
 
+template <typename T>
+struct is_graph_then_host<
```

Comment:
this is getting awfully close to just concatenating nouns in German :-) ... `is_then_host_graph_node` could have been an alternative but I don't feel strongly about it

Reply:
fine with me